tests/model_hub_tests/transformation_tests/generate_ref_diffs.py

```diff
@@ -43,9 +43,36 @@
 from openvino._offline_transformations import paged_attention_transformation
 from openvino._pyopenvino.op import _PagedAttentionExtension, Parameter, Result
 from optimum.intel import OVModelForCausalLM
+from optimum.intel.openvino import OVModelForVisualCausalLM
+from typing import Type, Union
 
 nodes_to_compare = ("ScaledDotProductAttention", "PagedAttentionExtension", "Parameter", "ReadValue", "Assign")
 
+def get_models_list_type(file_name: str, cls: Union[Type[OVModelForCausalLM], Type[OVModelForVisualCausalLM]]):
+    models = []
+    for line_items in utils.parse_list_file(file_name):
+        if len(line_items) == 2:
```

Comment:
minor: 
probably we can simplify it:

cnt_none_args = max(0, 4 - len(line_items))
model_name, model_link, mark, reason, *other = line_items + [None] * cnt_none_args

Reply:
let's leave it like that for now